vcs/vcs2vtk.py

```diff
@@ -2029,35 +2033,19 @@ def getProjectedBoundsForWorldCoords(wc, proj, subdiv=50):
     if vcs.elements['projection'][proj].type == 'linear':
         return wc
 
-    xs = numpy.linspace(wc[0], wc[1], subdiv).tolist()
-    xs += [wc[1], ] * subdiv
-    xs += numpy.linspace(wc[1], wc[0], subdiv).tolist()
-    xs += [wc[0], ] * subdiv
-
-    ys = [wc[2], ] * subdiv
-    ys += numpy.linspace(wc[2], wc[3], subdiv).tolist()
-    ys += [wc[3], ] * subdiv
-    ys += numpy.linspace(wc[3], wc[2], subdiv).tolist()
+    # the border in Cartesian space may not corespond to the border
+    # in the projected space.
```

Comment:
Given the change you've made to the code here, I don't understand this comment (or the title of the PR for that matter).  Not having strong numpy skills, it looks to me like you're doing the same thing as before, just in a more efficient manner: subdivide the bounding box, transform the subdivided points.

I may be reading it incorrectly, but the only thing that seems substantively different is that you filter the transformed points now, using the `setInfoToValid(...)` method, which has been slightly modified to allow for not having ghost points.  Is that correct?  Thanks in advance for straightening me out ;-)

Reply:
Indeed, there were two (related) problems: 1. some of the border points ended up inf in the projected space
2. Before we were projecting only the border, now I project the whole dataset. So before we were projecting 4*subdiv points, now we are projecting subdiv^2 points. Maybe I can do something smarter like reducing the bounds in the Cartesian but I'll have to know what are the max values I have to avoid. It may get tricker if data is not 0,360 but 60,420 for instance.

Reply:
Ah, gotcha.  In that case, it was my weak numpy-fu which failed me, and what you've done seems like a really good idea.  Now I need to check if we were able to merge the work I did which minimizes the number of times we call that method, since as you point out, it is now projecting many more points.